### ISSUE TITLE
Emoji 12

### ISSUE THREAD

**Issue by Madis0** [C1710/blobmoji#36]
https://blog.emojipedia.org/230-new-emojis-in-final-list-for-2019/

**Comment by C1710** [comment]
Most graphics are available on the new `emoji12`-branch (which might change, since I don't know if I want to include the redesigns I've done yet).  
Finishing everything will probably still take some days since it has been some time since I last added new emojis 😅...

Currently missing:
Skin tone variations, PNG images, new fonts (esp. the EmojiCompat font for which I haven't updated the emoji list yet).

**Comment by Madis0** [comment]
[Google's variants are now available.](https://blog.emojipedia.org/first-look-googles-2019-emoji-update/)

**Comment by Cubox-** [comment]
I checked a few of the new emojis, seeing the blob https://github.com/C1710/blobmoji/blob/c0f1a1df182843b4386a82b0385a01011159c6f8/svg/emoji_u1fa73.svg here made me smile!

**Comment by C1710** [comment]
The new font is now available - both the [usual font](https://github.com/C1710/blobmoji/blob/emoji12/emojicompat/NotoColorEmojiCompat.ttf) and the [EmojiCompat font](https://github.com/C1710/blobmoji/blob/emoji12/emojicompat/NotoColorEmojiCompat.ttf).  
However there will be some minor changes since I forgot to update the PNG files for some of the redesigned emojis; Emoji 12 should work...  
I also still need to update the `CHANGES.md` file before merging to master.

**Comment by simonsmh** [comment]
svg/emoji_u1f971.svg
![image](https://user-images.githubusercontent.com/9512067/57577623-caacd780-74ad-11e9-92b3-a253548affa6.png)
This entry seems not to be replaced at the PNG file.

**Comment by simonsmh** [comment]
Also is there any way to directly build the font from svg source? Currently it is still built with PNG files.

**Comment by C1710** [comment]
@simonsmh Oh, thanks for pointing that out 😄.  

Actually I don't know if the font can be build directly using the SVG files. However the color font standard used here only uses PNG, so I would say that you'd probably have to use another toolset for that.  
And last but not least, some emojis use features that are rather rare in SVG, like centered text placement for the 🔠 emoji.  
It might either look different than expected or it might not render at all (in fact, I even have issues opening the file outside of Inkscape. Affinity Designer even thinks the whole file is broken).

But I would move that discussion into another issue 😅

**Comment by C1710** [comment]
@simonsmh The PNG files are now fixed and the font is also updated.

**Comment by Madis0** [comment]
Some person-color sequences are not correctly applied, one-piece swimsuit is broken 🩱
https://emojipedia.org/emoji-12.0/

**Comment by C1710** [comment]
Well, the one piece swimsuit isn't broken, it's simply _missing_ (this was - of course - unintentional. I created all the files manually and apparently I missed it).  
But I don't know what you mean with 'to use a merged graphic'), could you please add a screenshot or something like that?

**Comment by C1710** [comment]
Nevermind, I found the issue(s):  
1. I forgot the single skin-colored version of People holding hands
2. At least for one-gendered pairs there is the rule that the left one's skin color is at least as dark as the right one's. This does not apply for mixed gender pairs (so, technically I already had these variants, but I deleted them because I thought they weren't in the official standard).

I'll probably fix these issues until Wednesday...
